handbook/benefits.md

```diff
@@ -441,7 +702,7 @@ When in the vicinity / metro area of a dbt Labs office, leverage the office inst
 
 #### What if I want to rent a different workspace for the day that is not available on liquidspace?
```

Comment:
For above, line 421/682 - change to "**dbt Labs'** remote-first.."

Reply:
Line 427/688 - the sentence doesn't really make sense to me. Suggestion:

We offer the following to enable employees to rent workspaces outside the home and optimize productivity:

Reply:
Line 434/695 - can we link Liquidspace?

Reply:
Line 438/699 -  dbt **Labs** office

Reply:
Can we capitalize Liquidspace in all instances?